library/core/src/fmt/mod.rs

```diff
@@ -323,6 +335,16 @@ impl<'a> ArgumentV1<'a> {
         unsafe { ArgumentV1 { formatter: mem::transmute(f), value: mem::transmute(x) } }
     }
 
+    arg_new!(new_display, Display);
+    arg_new!(new_debug, Debug);
+    arg_new!(new_octal, Octal);
+    arg_new!(new_lower_hex, LowerHex);
+    arg_new!(new_upper_hex, UpperHex);
+    arg_new!(new_pointer, Pointer);
+    arg_new!(new_binary, Binary);
+    arg_new!(new_lower_exp, LowerExp);
+    arg_new!(new_upper_exp, UpperExp);
```

Comment:
Can these be inlined into the format_args!() expansion?

Reply:
These are generic functions so I think yes.

Reply:
I actually meant can we inline their definition into the format_args!() expansion and omit those function definitions themself.